app/next-client-app/app/datasets/page.tsx

```diff
@@ -0,0 +1,85 @@
+import {
+  Breadcrumb,
+  BreadcrumbItem,
+  BreadcrumbLink,
+  BreadcrumbList,
+  BreadcrumbSeparator,
+} from "@/components/ui/breadcrumb";
+import { Button } from "@/components/ui/button";
+import { DataTable } from "@/components/data-table";
+import { columns } from "./columns";
+import { getDataSets } from "@/api/datasets";
+import { Tabs, TabsContent, TabsList, TabsTrigger } from "@/components/ui/tabs";
+import { objToQuery } from "@/lib/client-utils";
+import Link from "next/link";
+
+interface DataSetListProps {
+  searchParams?: { [key: string]: string | undefined } | {};
+}
+
+export default async function DataSets({ searchParams }: DataSetListProps) {
+  const query = objToQuery(searchParams ?? {});
+  const dataset = await getDataSets(query);
+
+  return (
+    <div className="pt-10 px-16">
+      <div>
+        <Breadcrumb>
+          <BreadcrumbList>
+            <BreadcrumbItem>
+              <BreadcrumbLink href="/">Home</BreadcrumbLink>
+            </BreadcrumbItem>
+            <BreadcrumbSeparator>/</BreadcrumbSeparator>
+            <BreadcrumbItem>
+              <BreadcrumbLink href="/datasets">Datasets</BreadcrumbLink>
+            </BreadcrumbItem>
+          </BreadcrumbList>
+        </Breadcrumb>
+      </div>
+      <div className="flex justify-between mt-3">
+        <h1 className="text-4xl font-semibold">Dataset List</h1>
+        <Link href="/">
+          <Button size="lg" className="text-md bg-[#475da7]">
```

Comment:
As discussed - would be great to drive these colours from tailwind themeing, instead of specifying it every time. 

https://tailwindcss.com/docs/customizing-colors#using-custom-colors

But set them on the default variant inside `Button.tsx`, so we just do it once.
There might be a couple of other places this is used, so fixing them in this PR would be great. (Just search for all instances of `#475da7`)

Reply:
The correct link is actually: https://tailwindcss.com/docs/customizing-colors#adding-additional-colors
so inside `tailwind.config.js`:
```
module.exports = {
  theme: {
    extend: {
      colors: {
        carrot: {
          ....
        },
      }
    },
```